### PR TITLE
Header/footer

### LINKED ISSUE #12: Header&Footer updated

### PR DESCRIPTION
**Pull request by jgrgurovic**

### CODE REVIEW

**Comment by ilovric00** [comment]
@jgrgurovic build is failing

File: tsconfig.json [hunk 1]
```diff
@@ -22,6 +22,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "views/homepage/OrganizationsSections"],
```
Comment (ilovric00) [inline tsconfig.json]: no

**Review by ilovric00 — CHANGES_REQUESTED** [review]
PR should contain only changes related to the task, in this case only header and footer, and here we have plenty of things mixed in one PR, mostly regarding auth

**Comment by jgrgurovic** [comment]
> @jgrgurovic build is failing

Build is fine now, not sure how the two pull requests got mixed together I suppose??

**Pull request merged**